internal/backend/grpc/router/resolve.go

```diff
@@ -25,6 +25,20 @@ type resolvedNodeInfo struct {
 	nodes []dns.Info
 }
 
+func (i resolvedNodeInfo) nodesOK() bool {
```

Comment:
I think this is wrong fix, as it goes "all or nothing", so if a mix of resolved/unresolved is treated as "unresolved"

I feel much easier fix is to make backend resolver return same record if not found (or wrap it to return same entry if not found)

Reply:
Reworked it a bit, pls have another look.